src/pipecat/services/openai.py

```diff
@@ -635,6 +635,9 @@ async def push_aggregation(self):
         self.reset()
 
         try:
+            if aggregation:
```

Comment:
Should we implement this same fix in the cases: 
- `GrokAssistantContextAggregator` 
- `GoogleAssistantContextAggregator`

Looking at the code it feels they would face the same issue.

Reply:
Ah, yes! Nice catch! Updated.